client/config/regenerate.js

```diff
@@ -5,34 +5,22 @@
  */
 var fs = require( 'fs' ),
 	path = require( 'path' ),
-	env = process.env.CALYPSO_ENV || 'development',
-	filename = env + '.json',
-	configPath = path.resolve( __dirname, '..', '..', 'config', filename ),
 	keysPath = path.resolve( __dirname, '..', '..', 'config', 'client.json' ),
-	data = JSON.parse( fs.readFileSync( configPath, 'utf8' ) ),
 	keys = JSON.parse( fs.readFileSync( keysPath, 'utf8' ) ),
-	enabledFeatures = process.env.ENABLE_FEATURES ? process.env.ENABLE_FEATURES.split(',') : [],
-	disabledFeatures = process.env.DISABLE_FEATURES ? process.env.DISABLE_FEATURES.split(',') : [],
+	data = require( 'config/reader' )( {
+		env: process.env.CALYPSO_ENV || 'development',
+		secrets: false
```

Comment:
I like that it's explicit (just in case something changes because we don't want to leak secrets) but we can probably catch that with tests.